lib/msal-node/src/network/HttpClient.ts

```diff
@@ -3,51 +3,61 @@
  * Licensed under the MIT License.
  */
 
-import { INetworkModule, NetworkRequestOptions } from '@azure/msal-common';
+import { INetworkModule, NetworkRequestOptions, NetworkResponse } from '@azure/msal-common';
 import { HttpMethod } from './../utils/Constants';
-import axios from 'axios';
+import axios, {AxiosRequestConfig} from 'axios';
 
 /**
- * This class implements the Fetch API for GET and POST requests. See more here: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+ * This class implements the API for network requests.
  */
 export class HttpClient implements INetworkModule {
+
     /**
-     * Axios CLient library for REST endpoints - Get request
+     * Http client for REST endpoints - Get request
      * @param url
-     * @param headers
-     * @param body
+     * @param options
      */
-    async sendGetRequestAsync<T>(url: string, options?: NetworkRequestOptions): Promise<T> {
-
-        // axios config
-        const request = {
-            method: HttpMethod.GET,
-            url: url,
-            headers: (options && options.headers)
+    async sendGetRequestAsync<T>(
+        url: string,
+        options?: NetworkRequestOptions
+    ): Promise<NetworkResponse<T>> {
+     const request: AxiosRequestConfig = {
+         method: HttpMethod.GET,
+         url: url,
+         headers: options && options.headers,
+         validateStatus: () => { return true }
```

Comment:
I believe you should be able to set this for all requests via the defaults object, which you can do in the constructor of this class.

Reply:
```js
axios.defaults.validateStatus = function () {
    return true;
};
```

Reply:
Yep, this works. Updated.